python/ray/data/_internal/plan.py

```diff
@@ -41,9 +41,12 @@
 class Stage:
     """Represents a Dataset transform stage (e.g., map or shuffle)."""
 
-    def __init__(self, name: str, num_blocks: Optional[int]):
+    def __init__(
+        self, name: str, num_blocks: Optional[int], run_by_consumer: bool = False
```

Comment:
```suggestion
        self, name: str, num_blocks: Optional[int], *, run_by_consumer: bool
```

Make ownership fields non-optional and keyword-only please, here and elsewhere.

Reply:
Done.